#1272 - Use eps by default in plots (#1303)
JuliaReach/LazySets.jl: docs/src/lib/interfaces.md

isbounded(::LazySet)
isbounded_unit_dimensions(::LazySet{N}) where {N<:Real}
an_element(::LazySet{N}) where {N<:Real}
tosimplehrep(::LazySet)
isuniversal(::LazySet{N}, ::Bool=false) where {N<:Real}
```

> The following functions dispatch in a general `LazySet` and work provided that the overapproximation using iterative refinement in two dimensions is available:

Reviewer: ```suggestion
The following functions work with general two-dimensional `LazySet`s, provided that the overapproximation using iterative refinement is available:
```
